### DISCUSSION TITLE
i get an error when use sample code source of date filter

### DISCUSSION THREAD

**Discussion by glonaldo70** [filamentphp/filament#14450] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

i try to use it in my code but not working. it show me 

Filter::make('created_at')
    ->form([
        DatePicker::make('created_from'),
        DatePicker::make('created_until'),
    ])
    ->query(function (Builder $query, array $data): Builder {
        return $query
            ->when(
                $data['created_from'],
                fn (Builder $query, $date): Builder => $query->whereDate('created_at', '>=', $date),
            )
            ->when(
                $data['created_until'],
                fn (Builder $query, $date): Builder => $query->whereDate('created_at', '<=', $date),
            );
    })

<img width="1353" alt="image" src="https://github.com/user-attachments/assets/f1b6e9cd-24a4-4f38-90bb-8f920241e6a4">

**Comment by leandrocfe** [comment, marked as answer]
try 
```php
->when(
    data_get($data, 'created_from'),
    ...
```

**Reply by glonaldo70** [reply]
Thank you, it's work
